Pipfile

```diff
@@ -4,9 +4,9 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
+tox = "*"
```

Comment:
why tox here?

Reply:
Fixed. Dropping for now py39 compatibility. Probably need update dependencies